src/band.rs

```diff
@@ -53,21 +55,21 @@ impl Band {
     /// #     0., 0., 0., 0., 0., 0., 0., 0., 0., 0.
     /// ], &[0.5, 1.5, 2.5]).unwrap();
     ///
-    /// let geojson_string = contours[0].to_geojson().to_string();
+    /// let geojson_string = contours[0].to_geojson().unwrap().to_string();
     ///
     /// assert_eq!(&geojson_string[0..27], r#"{"geometry":{"coordinates":"#);
     /// ```
-    pub fn to_geojson(&self) -> geojson::Feature {
+    pub fn to_geojson(&self) -> serde_json::Result<geojson::Feature> {
```

Comment:
I think we should use the `Result` type defined in error.rs (as it already implements `From<serde_json::error::Error> for Error` and since we have a `ErrorKind::JsonError`).
Or am I missing something that requires us to use `serde_json::Result` ?

Reply:
You're entirely correct, I forgot about that.